### ISSUE TITLE
Error: object 'worldpolys' not found

### ISSUE THREAD

**Issue by neonb88** [NovembreLab/Novembre_etal_2008_misc#1]
Hi,

I was trying to replicate your results and lack the code that generated them.  Please share the generating .R files

========================================================

Below is moot but left there for record-keeping reasons:

I was trying to replicate your results and everything was working until I ran `plot(worldpolys,xlim=c(-8,35),ylim=c(35,60),col=coldatamap[toupper(worldpolys$names),"color"])`, after which it gives me the following error:

===================================================

`Error in plot(worldpolys, xlim = c(-8, 35), ylim = c(35, 60), col = coldatamap[toupper(worldpolys$names),  : `

  `object 'worldpolys' not found`

===================================================

EDIT:  It might have something to do with this: `load("world_countries.rda'")` giving me the error message `world_countries' looks like a pre-2.4.0 S4 object: please recreate it`.  I was looking, but did not find an world_countries.r script or anything of the sort so I cannot recreate.  I understand R updating is not your fault, but your lab's reputation could be bolstered if Professor Pe'er and I can recreate your results.  Please update this

Warmly,
Nathan Bendich, working with Professor Itsik Pe er at Columbia University

**Comment by pcarbo** [comment]
@neonb88 See [here](https://github.com/NovembreLab/PCAviz/tree/master/inst/original-demos) for some runnable scripts based on this code. You might find these script easier to work with.

**Comment by neonb88** [comment]
@pcarbo Thanks!  I'm new to R.  Is there a particular function or script that replicates this paper's results?

**Comment by pcarbo** [comment]
@neonb88 I would start by trying to run the code in the `.R` files in the `R-3.2` folder.

**Comment by neonb88** [comment]
@pcarbo Hi, upon further reflection, it doesn't help me to have access to your plots.  Could I please see the R code that generates these results?  As long as I have all that code, I can stop pestering you.  Thanks, and godspeed

**Comment by pcarbo** [comment]
@neonb88 Did you look at files `demo.pcvsgeo.R`, `demo.pcvslatlong.R` or `demo.plotpc.R` following the link I included above? These files contain R code to generate plots.

**Comment by neonb88** [comment]
@pcarbo Perhaps I wasn't clear.  We're interested in how you got the results.  Thank you for the plotting, but with respect to the research, the plots are secondary to how the Novembre lab generated the data in the first place.

I would like the .R code that generated the data for **this** repository and made **these** plots: [1](https://github.com/NovembreLab/Novembre_etal_2008_misc/files/1795456/nature07331.2.pdf), and [2](https://github.com/NovembreLab/Novembre_etal_2008_misc/files/1795457/nature07331.3.pdf).  I can plot y'all's data using any language, like Matplotlib.  But I need the data first.  Do y'all hang on to that?

Thanks,
Nathan

**Comment by pcarbo** [comment]
@neonb88 You mean you would like to see how the principal components (PCs) were computed from the POPRES data?

**Comment by neonb88** [comment]
@pcarbo Yes.  Ah, now I've read this more carefully, and I see the README says "The raw POPRES data must be accessed via dbGAP."  Which version did y'all use?  From [here](https://www.ncbi.nlm.nih.gov/gap) ?

**Comment by pcarbo** [comment]
I suggest emailing John Novembre; he would be able to give you an answer to your question.

**Comment by sktrinh12** [comment]
> @pcarbo Yes. Ah, now I've read this more carefully, and I see the README says "The raw POPRES data must be accessed via dbGAP." Which version did y'all use? From [here](https://www.ncbi.nlm.nih.gov/gap) ?

Hi, @neonb88 were you able to get the raw data to re-produce the PCA plots? I was also hoping to re-produce them using python as well. Hope you can share some information. Thanks.
